lib/screens/home.dart

```diff
@@ -164,7 +164,10 @@ class HomeScreen extends StatefulWidget {
 
 class _HomeScreenState extends State<HomeScreen> with TickerProviderStateMixin {
   final _scaffoldKey = GlobalKey<ScaffoldState>();
-
+  late StreamingSharedPreferences streamPrefs;
+  String ttsChannel = '';
```

Comment:
Make this nullable and typed instead of the json string.

Reply:
Done